Upsert processing speed Increments (#45)
elasticio/salesforce-component-v2: README.md

* **success** - Boolean result of creation/update object
* **errors** - Arrey of errors if they exist

#### Known limitations

> If you add new field to object in Salesforce, you should restart flow to let them appear inside input metadata

Reviewer: ```suggestion
If you add a new field to an object in Salesforce, you must restart the flow to re-generate metadata
```
